paymaster/src/rpcMethods.ts

```diff
@@ -0,0 +1,54 @@
+import { BigNumberish, ethers } from 'ethers';
+import { paymasterAbi } from './abis';
+import { BytesLike } from '@ethersproject/bytes';
+import dotenv from 'dotenv';
+dotenv.config();
+
+export interface UserOperation {
+    sender: string
+    nonce: BigNumberish
+    initCode: BytesLike
+    callData: BytesLike
+    callGasLimit: BigNumberish
+    verificationGasLimit: BigNumberish
+    preVerificationGas: BigNumberish
+    maxFeePerGas: BigNumberish
+    maxPriorityFeePerGas: BigNumberish
+    paymasterAndData: BytesLike
+    signature: BytesLike
+}
+
+export async function sponsorTransaction(userOp: UserOperation): Promise<UserOperation> {
+    const paymasterAddress = process.env.PAYMASTER_ADDRESS as string;
+    const provider = new ethers.JsonRpcProvider(process.env.RPC_URL as string);
+    const signer = new ethers.Wallet(process.env.PRIVATE_KEY as string);
```

Comment:
We probably want to read all these values from the config instead of environment variables (then we can fill the config upstream from environment variable if desired) :)

I'm thinking probably we read those from a json file, and we generate the json file from `roll.py` based on values in `L2Config` (whose name needs to change)

Reply:
Okay, I see that you actually use the `.env` as the equivalent of the JSON file, so this is good actually, please ignore the comment :)